tests/test_algorithm.py

```diff
@@ -1819,6 +1821,29 @@ def test_empty_positions(self):
         self.assertTrue(all(num_positions == 0))
         self.assertTrue(all(amounts == 0))
 
+    @parameterized.expand([
+        ('noop_algo', noop_algo),
+        ('with_benchmark_set', set_benchmark_algo)]
+    )
+    def test_zero_trading_days(self, name, algocode):
+        """
+        Test that when we run a simulation with no trading days (e.g. beginning
+        and ending the same weekend), we don't crash on calculating the
+        benchmark
+        """
+        sim_params = copy(self.sim_params)
+        sim_params.period_start = pd.Timestamp('2006-01-14', tz='UTC')
```

Comment:
If we're setting all these attributes on the copied sim_params to new values, should we just make a new SimParams instance entirely?